CONTRIBUTING.rst

```diff
@@ -26,6 +28,12 @@ We recommend installing MLflow in its own conda environment for development, as
 You can verify that ``npm`` is on the PATH by running ``npm -v``, and
 `install npm <https://www.npmjs.com/get-npm>`_ if needed.
 
+If contributing to MLflow's R APIs, install `R <https://cloud.r-project.org/>`_. For changes to R
+documentation, also install `pandoc <https://pandoc.org/installing.html/>`_. If contributing
```

Comment:
This is an extreme nit, but I get a 404 when accessing `https://pandoc.org/installing.html/` with the trailing slash. `https://pandoc.org/installing.html` (no trailing slash) seems to work.